docs/source/API/core/parallel-dispatch/parallel_reduce.rst

```diff
@@ -0,0 +1,243 @@
+``parallel_reduce()``
+=====================
+
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage 
+-----
+
+.. code-block:: cpp
+
+    Kokkos::parallel_reduce( name, policy, functor, reducer... );
+    Kokkos::parallel_reduce( name, policy, functor, result...);
+    Kokkos::parallel_reduce( name, policy, functor);
+    Kokkos::parallel_reduce( policy, functor, reducer...);
+    Kokkos::parallel_reduce( policy, functor, result...);
+    Kokkos::parallel_reduce( policy, functor);
+
+Dispatches parallel work defined by ``functor`` according to the *ExecutionPolicy* and performs a reduction of the contributions provided by workers as defined by the execution policy. The optional label name is used by profiling and debugging tools. The reduction type is either a ``sum``, is defined by the ``reducer`` or is deduced from an optional ``join`` operator on the functor. The reduction result is stored in ``result``, or through the ``reducer`` handle. It is also provided to the ``functor.final()`` function if such a function exists. Multiple ``reducers`` can be used in a single ``parallel_reduce`` and thus, it is possible to compute the ``min`` and the ``max`` values in a single ``parallel_reduce``.
```

Comment:
For future work (not required for merging in this instance):  I believe `result` can be a sum, a min or a max